_stbt/ocr.py

```diff
@@ -363,6 +391,57 @@ def match_text(text, frame=None, region=Region.ALL,
     return result
 
 
+# Python 2.7 & 3.6 have `re._pattern_type` but that will be removed in Python
+# 3.7 where they introduce `re.Pattern`.
+PatternType = type(re.compile(""))
+
+
+def apply_ocr_corrections(text, corrections=None):
+    """Applies the same corrections as `stbt.ocr`'s ``corrections`` parameter.
+
+    This is also available as a separate function, so that you can use it to
+    post-process old test artifacts using new corrections. See also
+    `stbt.set_global_ocr_corrections`.
+    """
+    if corrections:
+        text = _apply_ocr_corrections(text, corrections)
+    if global_ocr_corrections:
+        text = _apply_ocr_corrections(text, global_ocr_corrections)
+    return text
+
+
+def _apply_ocr_corrections(text, corrections):
+    # Match plain strings at word boundaries:
+    pattern = "|".join(r"\b(" + re.escape(k) + r")\b"
+                       for k in corrections
+                       if isinstance(k, basestring))
+    if pattern:
+        replace = lambda matchobj: corrections[matchobj.group(0)]
+        text = re.sub(pattern, replace, text)
+    # Match regexes:
+    for k, v in corrections.items():
+        if isinstance(k, PatternType):
+            text = re.sub(k, v, text)
+    return text
+
+
+global_ocr_corrections = {}
+
+
+def set_global_ocr_corrections(corrections):
```

Comment:
An alternative to this function is  (and to config variables affecting `stbt` in general) is a module/object like `stbt.defaults`.  Then instead of configuring say match parameters in the config file you could do it in Python.  Something like:

```py
# Setup stbt defaults:
stbt.defaults.ocr_corrections = ...
stbt.defaults.match_parameters = ...
```

This would have the advantage that we could apply it to our other knobs more uniformly.  On the down-side it feels like there's more coupling here.

We can cross that bridge if/when we come to it though.